sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/StoreResult.java

```diff
@@ -42,6 +42,7 @@
     final public boolean isInvalidPartitionException;
     final public Uri storePhysicalAddress;
     final public boolean isThroughputControlRequestRateTooLargeException;
+    final public Double backendLatencyInMs;
```

Comment:
Any reason why `Double` instead of `double`? Other properties like requestCharge are not using boxed types.

Reply:
Yes, because default value of requestCharge is 0 in case of unexpected error. https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/StoreReader.java#L843
Which doesn't hold true for request latency - Don't want to confuse customers that we took 0 milliseconds :) 
That is the reason I used Double, so as to have null value in case of unexpected exception.

Reply:
Another thought on this, in case of unexpected errors, we can use -1 as value for request duration, but again, in my opinion null is better than -1, thoughts @mbhaskar ?

Reply:
null == data not present/set
I like that better than -1 which could be up for interpretation :-)

Reply:
Yes, that was my intention too @milismsft  :)

Reply:
So if `backendLatencyInMs` is empty, do we skip this in the diagnostics?

Reply:
No, we output it with null value.

Reply:
Thankyou